scripts/normalize-by-median.py

```diff
@@ -344,6 +344,11 @@ def main():  # pylint: disable=too-many-branches,too-many-statements
         else:
             output_name = args.single_output_file.name
         outfp = args.single_output_file
```

Comment:
The logic seems reasonably straightforward to me and I don't see how to simplify it easily.  I can't tell if I'm misreading it or if you're misreading it :). Help?

Also note that `.name` doesn't exist in certain cases that are hit by the tests. @bocajnotnef has been dealing with this by just applying str(args.single_output_file), which I guess works.